pkg/cli/admin/release/extract.go

```diff
@@ -174,6 +174,12 @@ func (o *ExtractOptions) Complete(f kcmdutil.Factory, cmd *cobra.Command, args [
 	}
 	o.From = args[0]
 
+	if err := o.SecurityOptions.Complete(f); err != nil {
+		return err
+	}
+	if !o.SecurityOptions.LookupClusterICSP && len(o.SecurityOptions.ICSPFile) == 0 {
+		o.SecurityOptions.TryAlternativeSources = true
```

Comment:
I don't understand, I thought that we want to check alternative sources when either cluster-icsp or icsp-file is passed?

Reply:
I'll rename that variable, that should be something like `TryAlternativeSourcesImplicitly` this is because for the `oc adm release` commands, the expectation is that oc should look in cluster without being askes.. but actually, I think this variable might not be necessary, I think I can do without it :eyes: will report back

Reply:
renamed to `ImplicitTryAlternativeSources` and set to true with `oc adm release extract` and `oc adm release info`

Reply:
Probably `AlwaysTryAlternativeSources` would be better and more explicit.

Reply:
updated to `AlwaysTryAlternativeSources`